.github/CODEOWNERS

```diff
@@ -1423,10 +1423,77 @@ x-pack/test/api_integration/deployment_agnostic/services/ @elastic/appex-qa
 x-pack/test/**/deployment_agnostic/ @elastic/appex-qa #temporarily to monitor tests migration
 
 # Core
-/test/plugin_functional/plugins/rendering_plugin @elastic/kibana-core
-/test/plugin_functional/plugins/session_notifications @elastic/kibana-core
+/x-pack/test/functional/page_objects/navigational_search.ts @elastic/kibana-core
+/x-pack/test/stack_functional_integration/apps/savedobjects_upgrade_testing @elastic/kibana-core
+/x-pack/test/functional/page_objects/status_page.ts @elastic/kibana-core
+/x-pack/test/functional/page_objects/share_saved_objects_to_space_page.ts @elastic/kibana-core
```

Comment:
Maaaybe `@elastic/kibana-security`, but perhaps it's OK for us to own this one too

Reply:
Could do joint ownership?  I could place their team after your team.  
Does that suffice?